etna/core/tests/test_middleware.py

```diff
@@ -0,0 +1,32 @@
+from unittest.mock import patch
+
+from django.test import TestCase, override_settings
+
+from ...ciim.utils import prevent_request_warnings
+
+
+@override_settings(
+    MAINTENANCE_MODE=True,
+    MAINTENENCE_MODE_ENDS="2011-11-04T00:05:23+04:00",
+)
+class TestMaintenanceMode(TestCase):
+    @prevent_request_warnings
+    def test_503_maintenance_mode_for_home_page(self):
+        response = self.client.get("/")
+        self.assertEquals(response.status_code, 503)
+
+
+@override_settings(
+    KONG_CLIENT_BASE_URL="https://kong.test",
+    MAINTENANCE_MODE=True,
+    MAINTENENCE_MODE_ALLOW_IPS="123.4.5.6",
```

Comment:
Isn't this setting value usually a list? https://github.com/nationalarchives/ds-wagtail/pull/381/files#diff-f7a16228131ae92457c22657bb33458207a7578b8cdb6ff796153916fa67a22dR323

Reply:
Set it to list of sting